app/dashboard/templates/toolbox.html

```diff
@@ -43,8 +43,19 @@ <h2 class="title">Toolbox</h2>
                 </div>
               </div>
             </div>
+
+            <div id="toc">
```

Comment:
We should consider removing `id` all together and using strictly class names. It allows for better .css inheritance if gitcoin decides to use the BEM method. Just my $0.02wei